pkgs/data/icons/candy-icons/default.nix

```
@@ -0,0 +1,31 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+}:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
}:
```
Small nit since this formatting is more popular in nixpkgs